### ISSUE TITLE
Post updated date is not tested

### ISSUE THREAD

**Issue by metamn** [WPTRT/theme-unit-test#18]
Currently the post entry meta tests only creation date and not updated date.

Like: "Posted on March 14, 2012" and not "Posted on March 14, 2012 and updated on  March 21, 2012".
Maybe this is an useful scenario ...

If you think I can add this feature.

**Comment by carolinan** [comment]
I agree that this would be useful, do you still have time to add it?
